Docs - Add draft whitelist sso doc (#1185)
CloudFire-LLC/cloudfire: docs/docs/user-guides/use-cases/nat-gateway/README.mdx

This guide will walk through a simple example restricting access for a
self-hosted web app to a single whitelisted static IP running Firezone.

This is commonly done in place of maintaining an IP whitelist for
multiple end users, which may become labor-intensive to manage as the

> access list grows.

Reviewer: ```suggestion
This is commonly done in place of maintaining an IP whitelist for
multiple team members, which becomes impossible to manage as the
access list grows and team members' IP addresses change.
```

a network proxy or NAT gateway to provide a single public egress IP for all the
devices connected to it.

In this example the protected resource and Firezone are

> in separate VPC regions.

Reviewer: ```suggestion
In this example the protected resource and Firezone are in separate VPC regions.
```

### Step 3 - Route traffic to the protected resource through the VPN server

> By default, all traffic from users will be routed through the VPN server

Reviewer: ```suggestion
By default, all traffic from team members will be routed through Firezone
```
